src/java/boa/aggregators/Aggregator.java

```diff
@@ -34,10 +35,12 @@
  */
 public abstract class Aggregator {
 	private long arg;
+	private String mlarg; //for ML
 	@SuppressWarnings("rawtypes")
 	private Context context;
 	private EmitKey key;
 	private boolean combining;
+	private int vectorSize; //for ML
```

Comment:
move to MLAggregator?

Reply:
I have moved all of them to MLAggregator.